server.js

```diff
@@ -11,9 +12,16 @@ app.get('/api', getEndpoints);
 app.get('/api/categories', getCatgories);
 app.get('/api/reviews/:review_id', getSingleReview);
 app.get('/api/reviews', getReviews);
+app.get('/api/reviews/:review_id/comments', getCommentsFromId);
 
 app.use('/*', (req, res) => {
         res.status(404).send({message:'404 path not found'})
 })
 
+app.use((err, req, res, next) => {
```

Comment:
You'll probably need to add in PSQL errors here - if you make a request and PSQL can't handle it then it'll spit out a specific PSQL error code e.g. 22P02 or similar.

Reply:
Sorry this is for the next ticket! But I'll leave it as a hint! :D